swarmcg/scoring/bonds.py

```diff
@@ -0,0 +1,107 @@
+import MDAnalysis as mda
+import numpy as np
+
+from swarmcg import config
+
+
```

Comment:
These 2 functions can be merged easily into a single one, I will do this soon. Same thing for angles and dihedrals.